pom.xml

```diff
@@ -133,7 +133,7 @@
         <scalatest.version>3.2.9</scalatest.version>
         <scopt.version>4.0.1</scopt.version>
         <slf4j.version>1.7.35</slf4j.version>
-        <log4j.version>2.17.1</log4j.version>
+        <log4j.version>2.17.2</log4j.version>
```

Comment:
hi @RishiKumarRay, you can add `<log4j.ignoreTCL>true</log4j.ignoreTCL>` in L1500 to fix this issue

Reply:
https://github.com/apache/spark/pull/35765 for more information